androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/BottomControls.kt

```diff
@@ -0,0 +1,41 @@
+package com.profusion.androidenhancedvideoplayer.components.playerOverlay
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.absolutePadding
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+
+private val BOTTOM_CONTROLS_HEIGHT = 50.dp
+
+@Composable
+fun BottomControls(
+    modifier: Modifier = Modifier,
+    isFullScreen: Boolean,
+    onFullScreenToggle: () -> Unit,
+    customization: ControlsCustomization
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(BOTTOM_CONTROLS_HEIGHT)
```

Comment:
I know that fixed sizes are not ideal, but in this case it is necessary to have a fixed size because of the weight property of the `MiddleContent` which tries to occupy all the remaining space available, so if the `BottomContent` does not have a height set, it will be "swallowed" by `MiddleContent`

Reply:
I used it this way because the `weight` property of MIddleContent was swallowing the composable BottomContent, but I found a way to do it without setting a fixed height. thanks for the comment.